components/Card/style.module.css

```diff
@@ -11,14 +12,15 @@
   text-decoration: none;
   border: 1px solid #eaeaea;
   border-radius: 10px;
-  transition: color 0.15s ease, border-color 0.15s ease;
+  transition: color 0.5s ease, background-color 0.5s ease, border-color 0.5s ease;
 }
 
-.card:hover,
 .card:focus,
-.card:active {
-  color: blue;
-  border-color: blue;
+.card:active,
+.card:hover {
+  color: white;
+  border-color: white;
+  background-color: orange;
```

Comment:
I like the upgrade to orange to make it more consist with our branding. However, I dislike the orange fill. Could you try orange as border color?

What about adding some animation effect on mouse hover, live rotating, shaking or something else?